src/ProgressOnderwijsUtils/Data/SqlCommandCreationContext.cs

```diff
@@ -12,10 +12,11 @@ public interface IHasDefaultCommandTimeout
 
 public sealed class SqlConnectionContext : SiteBase, IAttachedToTracer, IHasDefaultCommandTimeout
 {
-    public SqlConnectionContext(ISqlCommandTracer tracer, CommandTimeoutDefaults timeoutDefaults)
+    public SqlConnectionContext(ISqlCommandTracer tracer, CommandTimeoutDefaults timeoutDefaults, string? name = null)
     {
         Tracer = tracer;
         TimeoutDefaults = timeoutDefaults;
+        Name = name;
```

Comment:
Not a huge fan of optional args, and for constructors especially so since you can already do `new(tracer, timeouts) { Name = "foobar" }`